src/stores/BoxesStore.ts

```diff
@@ -65,6 +66,8 @@ export class BoxesStore {
 		},
 	];
 
+	isSelectedBoxValid: boolean = true;
```

Comment:
I suppose we need to set it true when selectedBox changed

Reply:
isSelectedBoxValid is updated at the same time with monaco editors content validation, which occurs when the box is changed. 
Why we should update it twice? Or it might be better not to initialize this? For example: `isSelectedBoxValid ?: boolean;` ?